.editorconfig

```diff
@@ -135,9 +144,27 @@ dotnet_diagnostic.IDE0011.severity = silent
 # Object initialization can be simplified / Use object initializer.
 dotnet_diagnostic.IDE0017.severity = warning
 
+# Inline variable declaration.
+dotnet_diagnostic.IDE0018.severity = warning
+
+# Use pattern matching to avoid 'as' followed by a 'null' check.
+dotnet_diagnostic.IDE0019.severity = warning
+
+# Use pattern matching to avoid 'is' check followed by a cast.
+dotnet_diagnostic.IDE0020.severity = warning
+
 # Collection initialization can be simplified
 dotnet_diagnostic.IDE0028.severity = warning
 
+# Use coalesce expression (non-nullable types).
+dotnet_diagnostic.IDE0029.severity = warning
+
+# Use coalesce expression (nullable types).
+dotnet_diagnostic.IDE0030.severity = warning
+
+# Use explicitly provided tuple name.
```

Comment:
```suggestion
# Prefer explicitly provided tuple element name.
```

Reply:
I've been copy pasting the titles of the docs, e.g. https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0033. Don't feel like changing the approach much for one item.

Reply:
Weird. I commented because that's not the title VSCode gave me 🤷‍♂️   Alright then, never mind.